sw/device/tests/aon_timer_irq_test.c

```diff
@@ -130,20 +132,20 @@ static void execute_test(dif_aon_timer_t *aon_timer, uint64_t irq_time_us,
   uint32_t start_tick = (uint32_t)tick_count_get();
 
   ATOMIC_WAIT_FOR_INTERRUPT(peripheral ==
-                            kTopEarlgreyPlicPeripheralAonTimerAon);
+                            dt_aon_timer_instance_id(kAonTimerDt));
```

Comment:
I didn't notice this on the earlier PR, but what was the motivation for using pointers to the table?